tests/third_party/cupy/core_tests/test_ndarray_complex_ops.py

```diff
@@ -47,7 +47,6 @@ def test_angle(self, xp, dtype):
         return xp.angle(x)
 
 
-@testing.gpu
```

Comment:
Can we extend parametrize scope in `tests/third_party/cupy/math_tests/test_arithmetic.py` with `real` and `imag` as done in cupy?

Reply:
Added some of them. There a few other tests that are not added since they use the fact the `numpy.real` and `numpy.imag` return a view of the original array while in current implementation of `dpnp`, a copy is returned.